crates/astria-composer/src/searcher/executor.rs

```diff
@@ -84,6 +86,7 @@ impl Executor {
     pub(super) fn new(
         sequencer_url: &str,
         private_key: &SecretString,
+        transaction_resubmission_tx: mpsc::Sender<Vec<Action>>,
```

Comment:
This seems like a code smell. If you need a queue of some sort, it's better to use `FuturesUnordered` (or probably maybe `FuturesOrdered`).